packages/esm-patient-attachments-app/src/attachments/image-preview.scss

```diff
@@ -26,6 +26,16 @@
   color: $ui-01;
 }
 
+.closePreviewButton  {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  height: 40px;
+  width: 40px;
```

Comment:
Hi @usamaidrsk !
Since we are using the Carbon Buttons, we will have to look into why the carbon component is breaking, instead of passing a custom `width` and `height`

Reply:
I have updated this to use the button size prop. 
Thanks